config/src/aws-net-std__ipv6.adb

```diff
@@ -88,6 +88,14 @@ package body AWS.Net.Std is
    --  Returns the inet address information for the given host and port.
    --  Flags should be used from getaddrinfo C routine.
 
+   function Choose_IPv6
+     (Chain : OS_Lib.Addr_Info_Access) return OS_Lib.Addr_Info_Access;
```

Comment:
Looking at the code, it seems that the Chain and return value should never be null, right? If so:

  (Chain : not null OS_Lib.Addr_Info_Access) return not null OS_Lib.Addr_Info_Access;

Reply:
done